### PR TITLE
Update travis

### PR DESCRIPTION
**Pull request by antgonza**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -10,35 +10,46 @@ before_install:
...
+  - pip install sphinx sphinx-bootstrap-theme coveralls
   - pip install https://github.com/biocore/qiita/archive/master.zip --process-dependency-links
-  # installing other deps
-  - conda install --yes -c jjhelmus glpk
-  - conda install --yes -c bioconda diamond bowtie2 trimmomatic=0.36 fastqc=0.11.5
-  #
-  - export MOI_CONFIG_FP=$PWD/miniconda3/envs/qiita_env/lib/python2.7/site-packages/qiita_core/support_files/config_test.cfg
-  - ipython profile create qiita-general --parallel
-  - qiita-env start_cluster qiita-general
+  - sed 's/PORT = 7777/PORT = 6379/g' ${PWD}/miniconda3/envs/qiita/lib/python2.7/site-packages/qiita_core/support_files/config_test.cfg > config_test.cfg
```
Comment (tanaes) [inline .travis.yml:20]: Is this a travis-specific port conflict?

**Review by tanaes — APPROVED** [review]
Looks good to me!

**Comment by tanaes** [comment]
...although, it looks like the humann2 tests are erroring now. maybe Python3 issue? Do we want to fix for now?

File: .travis.yml (hunk 1 above)
Comment (semarpetrus) [reply .travis.yml:20]: From my experience, 6379 is the default port for redis server.

Comment (antgonza) [reply .travis.yml:20]: right! the qiita main travis install has 2 redis servers (main: 7777 and redbiom: default) so here we are copying the config file and making the main qiita redis server 7777

**Comment by antgonza** [comment]
I fixed the python3 sorting errors, the new one is
```The database file for MetaPhlAn does not exist at /home/travis/build/qiita-spots/qp-shotgun/miniconda3/envs/qp-shotgun/bin/db_v20/mpa_v20_m200.pkl . Please provide the location with --metaphlan-options```. Note that the latests version doesn't come with a DB (db_v20) so it's not being copied. Do you know where can we get it?

**Comment by tanaes** [comment]
ah this is the bigass metaphlan db. I've made a streamlined mini db for
testing here:
https://github.com/biocore/oecophylla/blob/dev/test_data/test_dbs/mp2_mini.tar.gz

small enough to include in test files (3.9 MB)

**Comment by antgonza** [comment]
closing in favor of: https://github.com/qiita-spots/qp-shotgun/pull/33

**Pull request closed without merging**